### PR TITLE
Add 'OpenStack deployment with DevStack' tutorial

### PR DESCRIPTION
**Pull request by alexcoman**

### CODE REVIEW

File: _posts/tutorial/openstack/2017-05-15-instalare-openstack-folosind-devstack.markdown [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+### Modulul SELinux
+
+Pentru a ușura pregătirea mediului o să începem prin a dezactiva modulul **SELinux**(*Security-Enhanced Linux*)  din cadrul nucleului de Linux.
+
+```bash
+~ $ echo "SELINUX=disabled" > /etc/sysconfig/selinux
+~ $ echo "SELINUXTYPE=minimum" >> /etc/sysconfig/selinux
+```
+**Avertisment**: acest lucru va scadea securitatea sistemului de operare.
```
Comment (mateimicu) [inline _posts/tutorial/openstack/2017-05-15-instalare-openstack-folosind-devstack.markdown]: sc**ă**dea

File: _posts/tutorial/openstack/2017-05-15-instalare-openstack-folosind-devstack.markdown [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+```
+
+În cazul în care conexiunile respective nu sunt active putem folosi comanda `nmcli con up <nume_conexiune>`.
+
+```
+# De exemplu
+~ $ nmcli con up eth1
+```
+
+Pasul următor este să configurăm static interfața pe care o să expunem interfața web.
```
Comment (mateimicu) [inline _posts/tutorial/openstack/2017-05-15-instalare-openstack-folosind-devstack.markdown]: Aici vorbesti de `interfața` si cand te referi la NIC si cand te referi la horizon si e putin confusing.
Maybe:
```
Pasul .....**prin** care o să expunem **Horizon**-> de la horizon link cartre ce e 
```

**Review by mateimicu — CHANGES_REQUESTED** [review]

File: _posts/tutorial/openstack/2017-05-15-instalare-openstack-folosind-devstack.markdown (hunk 1 above)
Comment (alexcoman) [reply _posts/tutorial/openstack/2017-05-15-instalare-openstack-folosind-devstack.markdown]: Done.

File: _posts/tutorial/openstack/2017-05-15-instalare-openstack-folosind-devstack.markdown (hunk 2 above)
Comment (alexcoman) [reply _posts/tutorial/openstack/2017-05-15-instalare-openstack-folosind-devstack.markdown]: Done.

**Pull request merged**